tutorials/optForceGAMS/optForceTutorialGAMS.m

```diff
@@ -0,0 +1,204 @@
+%% OptForce Tutorial
+% Sebasti�n N. Mendoza F. May 30th, 2017. snmendo@uc.cl
+
+% In this tutorial we will run optForce. For a detailed description of the
+% procedure, please see[1]: Ranganathan S, Suthers PF, Maranas CD (2010)
+% OptForce: An Optimization Procedure for Identifying All Genetic
+% Manipulations Leading to Targeted Overproductions. PLOS Computational
+% Biology 6(4): e1000744. https://doi.org/10.1371/journal.pcbi.1000744.
+
+% Briefly, the problem is to find a set of interventions of size "K" such
+% that when these interventions are applied to a wild-type strain, the
+% mutant created will produce a particular target of interest in a higher
+% rate than the wild-type strain. The interventions could be knockouts
+% (lead to zero the flux for a particular reaction), upregulations(increase
+% the flux for a particular reaction) and downregulations (decrease the
+% flux for a particular reaction). 
+
+% This prodecure consists on three steps:
+% 1) Perform FVA in both wild type and mutant strains 
+% 2) Find the must sets, i.e, reactions that MUST increase or decrease
+% their flux in order to achieve the phenotype in the mutant strain.
+% 3) Find the interventions needed that will ensure a increased production
+% of the target of interest 
+
+% For example, imagine that we would like to increase the production of
+% succinate in Escherichia coli. Which are the interventions needed to
+% increase the production of succinate? We will approach this problem in
+% this tutorial and we will see how each of the three steps are solved.
+
+% In this tutorial we will run steps 2 and 3 in GAMS and we will use MATLAB
+% to call GAMS. Therefore, GAMS must be intalled in the system before
+% running this turorial. After GAMS installation, we have to add tha path
+% where GAMS in located
+initCobraToolbox
+
+gamsPath = 'C:\GAMS\win64\24.8';
```

Comment:
Ok. I will change it. After modifying the function, I have to generate a new pull request?